workload/resources/EmotionAnalysisWebNNExecutor.js

```diff
@@ -23,14 +23,31 @@ class EmotionAnalysisWebNNExecutor extends WebNNExecutor {
   };
 
   /** @override */
-  _loadModel = async (modelId, coModelId) => {
+  doInitialRunner = (modelId, coModelId) => {
```

Comment:
Frankly, I prefer to use `initializeRunner` rather than `doInitialRunner`, and hate `do` here. But I know you've submitted this name to examples.

Reply:
Thanks @ibelem. 
It's a good suggestion, for later enhancement example to also involve workload function, I will optimize that.